qemu/tests/x86_cpu_flags.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from avocado.utils import cpu
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flags.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, boot guest with the cpu flags
+    3) Check cpu flags in guest(only for linux guest)
+    4) Reboot guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vendor_id = params.get("vendor_id", "")
+    if vendor_id:
+        if vendor_id != cpu.get_vendor():
+            test.cancel("Host vendor %s does not support this test case" % vendor_id)
```

Comment:
I guess here should be `cpu.get_vendor()` instead of `vendor_id`.

Reply:
@nanliu-r Or you might say: `This test case only support {vendor_id}`.

Reply:
> @nanliu-r Or you might say: `This test case only support {vendor_id}`.

Yes, sorry for so much confusing and have update, thanks.